src/index.js

```diff
@@ -82,7 +83,16 @@ function configureInterceptors(instance, url, { username, password } = {}) {
 
   instance.interceptors.response.use(res => {
     return res.data;
+  }, err => {
+    if (!(err instanceof Error) && typeof err === "object") {
+      return Promise.reject(errorify(err));
+    }
+    return Promise.reject(err);
```

Comment:
Okay I see.  So currently I'm just checking if what's returned from `auth` is a valid `Error`, and if it _isn't_ I then convert it to an `HttpError`.  If it _was_ a valid `Error` then just pass it back as is.

So we could update line 90 to be: `return Promise.reject(errorify(err))` and that would mean *all* errors returned from `authDriver` are properly specced http errors?

Is that what you mean?